auction/contracts/AuctionHouse.sol

```diff
@@ -431,5 +435,10 @@ contract AuctionHouse is AuctionHouseBase, InternalTransferExecutor {
         return transferManager;
     }
 
+    function changeMinimalDuration(uint newValue) external onlyOwner {
+        emit MinimalDurationChanged(minimalDuration, newValue);
+        minimalDuration = newValue;
```

Comment:
1.How do You think, we need to check newValue > 0?, or the main idea is: to set minimalDuration==0, and after that it is possible to set aucData.duration==0?
2.can we call emit as a final doing? like that:
 minimalDuration = newValue;

 emit MinimalDurationChanged(minimalDuration, newValue);

Reply:
1. yes we want to have aucData.duration==0 for tests
2. that way the same values are gonna be emitted